.travis.yml

```diff
@@ -50,19 +50,19 @@ before_install:
     - go get -u github.com/axw/gocov/gocov
     - go get -u github.com/mattn/goveralls
     - go get -u golang.org/x/tools/cmd/cover
-    - go get -u github.com/fzipp/gocyclo
 
     # Install godep for dependency management
     - go get -u github.com/tools/godep
 
     # Merge all subpackage code coverage reports into a single report
     - go get -u github.com/modocache/gover
 
-    # Install varcheck
-    - go get -u github.com/opennota/check/cmd/varcheck
+    # Get tools used in Makefile
+    - make get-tools
 
-    # Install deadcode
-    - go get -u github.com/mendersoftware/deadcode
+install:
+    # Get dependencies
+    - go get -t ./...
```

Comment:
Not really. In my opinion, Travis buils are bad in the sense that we're not using dependencies vendored by godep (ones in `Godeps/_workspace`). This happens as a result of 2 issues. Firstly we were directly calling `go build ...` instead of `godep go build ..`. Secondly (a side-effect of the first one), Travis automatically runs `go get -t ./...` if there's no install script, hence external dependencies were downloaded into `GOPATH`, each directly from `master` branch. Since the builds were never run with `go build .. -v`, it's not really easy to spot which packages are used.